src/Symfony/Contracts/Cache/ExtendedTagAwareCacheInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Contracts\Cache;
+
+use Psr\Cache\CacheItemPoolInterface;
+
+/**
+ * An interface that extends TawAwareCacheInterface with the methods from PSR-6's CacheItemPoolInterface.
```

Comment:
There's a typo: `Taw` => `Tag`, also present in the PR description.

Reply:
fixed thanks